src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+$card-width: 200px;
+$card-height: 406px;
+$image-width: $card-width - 40px;
+$image-height: $card-height - 272px;
```

Comment:
The variables `$image-width` and `$image-height` are defined but not used in the `.image` class. Instead, the same calculations are repeated directly in the `.image` class. Consider using these variables to maintain consistency and improve readability.